tests/bluetooth/tester/src/btp_bap.c

```diff
@@ -2535,6 +2539,46 @@ static uint8_t pacs_update_characteristic(const void *cmd, uint16_t cmd_len,
 	return BTP_STATUS_SUCCESS;
 }
 
+static uint8_t pacs_set_location(const void *cmd, uint16_t cmd_len,
+				 void *rsp, uint16_t *rsp_len)
+{
+	const struct btp_pacs_set_location_cmd *cp = cmd;
+	int err = bt_pacs_set_location((enum bt_audio_dir)cp->dir,
+				       (enum bt_audio_location)cp->location);
+
```

Comment:
```suggestion
	const int err = bt_pacs_set_location((enum bt_audio_dir)cp->dir,
				             (enum bt_audio_location)cp->location);

```

or

```suggestion
	int err;
	
	err = bt_pacs_set_location((enum bt_audio_dir)cp->dir,
				   (enum bt_audio_location)cp->location);

```
for consistency

Reply:
Done.